src/plugins/trigger/trigger-buffer.h

```diff
@@ -22,7 +22,7 @@
 
 #define TRIGGER_BUFFER_NAME "monitor"
 
-struct t_gui_buffer *trigger_buffer;
```

Comment:
Your undo is on file xfer-config.h but not this file (trigger-buffer.h).
And here I'm commenting on the "Files changed" tab in pull request, so this includes all your commits.